### PR TITLE
Book storage

### PR DESCRIPTION
**Pull request by SashaNehai**

### CODE REVIEW

File: KotBook/KotBook/Book.swift [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+//
+//  Created by Александр Нехай on 12/25/19.
+//  Copyright © 2019 kotiki. All rights reserved.
+//
+
+import Foundation
+
+class Book {
+	
+	var bookName: String?
```
Comment (evilutioner) [inline KotBook/KotBook/Book.swift]: please, rename _bookName_ -> _name_

File: KotBook/KotBook/Book.swift [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	var bookPath: String?
+	var bookAuthor: String?
+	
+	//	TODO: Create func that find (get) name of the book (API get or ect.)
+	
+	//	MARK: Method required for find File Path in app. Need to update if the directory of stored files will be updated.
```
Comment (evilutioner) [inline KotBook/KotBook/Book.swift]: better way to do it like  "/// comment"

File: KotBook/KotBook/Book.swift (hunk 1 above)
Comment (SashaNehai) [reply KotBook/KotBook/Book.swift]: done

File: KotBook/KotBook/Book.swift (hunk 2 above)
Comment (SashaNehai) [reply KotBook/KotBook/Book.swift]: done

**Pull request merged**